<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>433</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>일반회계 수입 지출 주간보고 이메일발송</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>107</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>받는사람 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>140</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>참조:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_email_Button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>61</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>보내기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="email_close_Button">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>61</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>180</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>제목 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="subject_widget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>431</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="body_contents_widget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>277</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>287</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>내용 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>첨부파일 :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="receiver_name_combo">
   <property name="geometry">
    <rect>
     <x>93</x>
     <y>106</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="co_receiver_name_combo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_line_edit_To">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>106</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_line_edit_Cc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="file_select_Button">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>220</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>파일선택</string>
   </property>
  </widget>
  <widget class="QListWidget" name="attached_file_listWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>send_email_Button</tabstop>
  <tabstop>email_close_Button</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>email_close_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>email_send_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>510</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>send_email_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>email_data_collect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>file_select_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>open_file_dialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>527</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>email_send_close()</slot>
  <slot>email_data_collect()</slot>
  <slot>open_file_dialog()</slot>
 </slots>
</ui>
